challenge_1/go/shivams/reverse.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"fmt"
+	//"strings"
+)
+
+func reverseString(str string) string {
+	var revStr string
+	for i := len(str) - 1; i >= 0; i-- {
+		revStr = revStr + string(str[i])
```

Comment:
This will cause a lot of memory churn as it copies revStr to a new instance for each character this appends to the end.